cifs/parse_cifs_test.go

```diff
@@ -0,0 +1,362 @@
+Locks: 0 sent 0 failed
+IOCTLs: 0 sent 0 failed
+Cancels: 0 sent 0 failed
+Echos: 0 sent 0 failed
+QueryDirectories: 0 sent 0 failed
+ChangeNotifies: 0 sent 0 failed
+QueryInfos: 0 sent 0 failed
+SetInfos: 0 sent 0 failed
+OplockBreaks: 0 sent 0 failed`,
+			stats: &cifs.ClientStats{
+				Header: map[string]uint64{
+					"operations":         0,
+					"sessionCount":       0,
+					"sessions":           2,
+					"shareReconnects":    0,
+					"shares":             4,
+					"smbBuffer":          2,
+					"smbPoolSize":        6,
+					"smbSmallBuffer":     2,
+					"smbSmallPoolSize":   30,
+					"totalMaxOperations": 2,
+					"totalOperations":    90,
+				},
+				SMBStatsList: []*cifs.SMBStats{
+					&cifs.SMBStats{
+						SessionIDs: cifs.SessionIDs{
+							SessionID: 1,
+							Server:    "server",
+							Share:     "\\share1",
+						},
+						Stats: map[string]uint64{
+							"cancelsSent":            0,
+							"cancelsFailed":          0,
+							"changeNotifiesSent":     0,
+							"changeNotifiesFailed":   0,
+							"closesSent":             0,
+							"closesFailed":           0,
+							"createsSent":            0,
+							"createsFailed":          2,
+							"echosSent":              0,
+							"echosFailed":            0,
+							"flushesSent":            0,
+							"flushesFailed":          0,
+							"ioCTLsSent":             0,
+							"ioCTLsFailed":           0,
+							"locksSent":              0,
+							"locksFailed":            0,
+							"logoffsSent":            0,
+							"logoffsFailed":          0,
+							"negotiatesSent":         0,
+							"negotiatesFailed":       0,
+							"oplockBreaksSent":       0,
+							"oplockBreaksFailed":     0,
+							"queryDirectoriesSent":   0,
+							"queryDirectoriesFailed": 0,
+							"queryInfosSent":         0,
+							"queryInfosFailed":       0,
+							"readsSent":              0,
+							"readsFailed":            0,
+							"sessionSetupsSent":      0,
+							"sessionSetupsFailed":    0,
+							"setInfosSent":           0,
+							"setInfosFailed":         0,
+							"treeConnectsSent":       0,
+							"treeConnectsFailed":     0,
+							"treeDisconnectsSent":    0,
+							"treeDisconnectsFailed":  0,
+							"writesSent":             0,
+							"writesFailed":           0,
+							"smbs":                   20,
+						},
+					},
+				},
+			},
+		}, {
+			name: "Mixed statistics (SMB1 then SMB2)",
+			content: `Resources in use
+CIFS Session: 1
+Share (unique mount targets): 2
+SMB Request/Response Buffer: 1 Pool size: 5
+SMB Small Req/Resp Buffer: 1 Pool size: 30
+Operations (MIDs): 0
+
+0 session 0 share reconnects
+Total vfs operations: 16 maximum at one time: 2
+
+1) \\server1\share1
+SMBs: 9 Oplocks breaks: 0
+Reads:  0 Bytes: 0
+Writes: 0 Bytes: 0
+Flushes: 0
+Locks: 0 HardLinks: 0 Symlinks: 0
+Opens: 0 Closes: 0 Deletes: 0
+Posix Opens: 0 Posix Mkdirs: 0
+Mkdirs: 0 Rmdirs: 0
+Renames: 0 T2 Renames 0
+FindFirst: 1 FNext 0 FClose 0
+
+2) \\server2\share2
```

Comment:
Can you use different numbers in the second fixture? I think there might be an issue with the parsing.